app/models/parser/twitter_search_item.rb

```diff
@@ -0,0 +1,36 @@
+module Parser
+  class TwitterSearchItem < Base
+    include ProviderTwitter
+
+    TWITTER_SEARCH_ITEM_URL = /https:\/\/twitter\.com\/search[^\s"]*/
+
+    class << self
+      def type
+        'twitter_search_item'.freeze
+      end
+
+      def patterns
+        [TWITTER_SEARCH_ITEM_URL]
+      end
+    end
+
+    private
+
+    # Main function for class
+    def parse_data_for_parser(_doc, _original_url, _jsonld_array)
+      handle_exceptions(StandardError) do
+        params = URI.parse(url).query.split('&').map { |pair| pair.split('=') }.to_h
+        src = params['src']
+        search_term = params['q']
+
+        @parsed_data.merge!(
+          title: search_term,
+          search_term: search_term,
+          src: src,
```

Comment:
Question: I wonder about adding new keys. When I was making an update for the tiktok parser Christa made this [comment](https://github.com/meedan/pender/pull/367#issuecomment-1642381056). What do you guys think?

Reply:
Good point! For consistency, I think it could go under a `raw` key, which is where we usually put parser-specific response. What do you think @vasconsaurus ?

Reply:
Thank you, and yes that is a better place to have it. I will move `src` to `raw`.